noxfile.py

```diff
@@ -73,10 +77,13 @@ def test_torch_nightly(session):
     session.install(
         *COMMON_TEST_DEPENDENCIES,
         ONNX_RUNTIME,
+        TRANSFORMERS,
     )
     session.install("-r", "requirements/ci/requirements-onnx-weekly.txt")
     session.install("-r", "requirements/ci/requirements-pytorch-nightly.txt")
     session.install(".", "--no-deps")
+    if sys.platform == "linux":
+        session.install("", "onnxruntime-training")
```

Comment:
I would call this out in the docstring under `test_torch_nightly` to make it very explicit what we are doing.

Reply:
done